<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncoderForm</class>
 <widget class="QWidget" name="EncoderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>226</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>467</width>
    <height>226</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>467</width>
    <height>226</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encoder</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLineEdit" name="lineString">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="encodeButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioHex">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioChar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="isUrlencoded">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Url-encoded</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MSSQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MySQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Base64</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="isPlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Play with letters</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="decodeButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
